0-module-and-module-source.emu

```diff
@@ -548,8 +550,11 @@ location: https://tc39.es/proposal-compartments/
                 1. Else, if Type(_importMeta_) is not Object, then
                   1. Throw a *TypeError* exception.
                 1. Perform ? RequireInternalSlot(_moduleSource_, [[ModuleSource]]).
+                1. Let _evalRealm_ be the current Realm Record.
```

Comment:
Does this mean the [[Realm]] from the current Execution Context?

My feeling is that we should be getting [[Realm]] from an internal slot of the %Module% constructor, patterned after %Function%.

Reply:
Yes, this is the pattern used by Function constructor, which just calls https://tc39.es/ecma262/#sec-createdynamicfunction, and this abstract operation gets the current realm.